.ci_support/migrations/cuda120.yaml

```diff
@@ -3,7 +3,7 @@ __migrator:
   kind:
     version
   migration_number:
-    1
+    3
```

Comment:
No change to this file does anything as long as you don't specify `use_local: true` (which you shouldn't). Based on the name of presence of a given migrator file (by file name), conda smithy will always prefer the most current version of that migration in the global pinning.

Reply:
If this file doesn't get updated comida smithy rerender was failing.

Reply:
Hm, strange. Can you try just updating the `migration_number` and nothing else?